cmd/antrea-agent/options.go

```diff
@@ -31,9 +31,10 @@ import (
 
 const (
 	defaultOVSBridge          = "br-int"
-	defaultHostGateway        = "gw0"
+	defaultHostGateway        = "antrea-gw0"
 	defaultHostProcPathPrefix = "/host"
 	defaultServiceCIDR        = "10.96.0.0/12"
+	defaultTunnelType         = ovsconfig.GeneveTunnel
```

Comment:
So that wasn't actually done in https://github.com/vmware-tanzu/antrea/pull/858?

Reply:
Right. It turned out #858 missed this change to set the tunnel type to Geneve when user does not populate the tunnel type option.
Anyway, I removed this change from this PR, as it seems the reason of netpol test failure.